where-to-travel/src/main/webapp/signup.js

```diff
@@ -77,17 +77,16 @@ $('#signUp-modal').on('hidden.bs.modal', function(){
  
 /** Signs up user using Firebase API **/
 function signUp() {
-  const promise = firebase.auth().createUserWithEmailAndPassword(emailSignUp.value, passwordSignUp.value);
-  promise.then(_ => {      
-    // Add user information to the real time database in Firebase
-    let ref = firebase.database().ref('users');
-    let data = {
-      name: nameSignUp.value,
-      email: emailSignUp.value,    
-      uID: firebase.auth().currentUser.uid
-    };
-    ref.push(data)
+  const promise = firebase().auth.createUserWithEmailAndPassword(emailSignUp.value, passwordSignUp.value);
+    promise.then(_ => {
```

Comment:
Rename promise to something more descriptive

Reply:
+1